src/commands/airdrop/index/processor.ts

```diff
@@ -262,7 +263,7 @@ export async function handleAirdrop(
 ) {
   const payload = await processor.getAirdropPayload(msgOrInteraction, args)
   const { amount, token, all } = payload
-  const { balance, usdBalance } = await validateBalance({
+  const { balance } = await validateBalance({
```

Comment:
why not use usdBalance

Reply:
It return 0
![Screen Shot 2023-04-04 at 2 14 11 PM](https://user-images.githubusercontent.com/39359294/229716235-d9087f49-18a8-4308-8d51-e0bcca7758a6.png)

Reply:
need to check the logic in validateBalance to see why it return 0